### PR TITLE
feat: add video export

### PR DESCRIPTION
**Pull request by Jonas1312**

### CODE REVIEW

File: tests/e2e/test_copy_project.py [hunk 1]
```diff
@@ -395,8 +396,7 @@ def test_copy_project_e2e(kili, src_project):
         for asset_src, asset_new in zip(assets_src, assets_new):
             assert asset_src["externalId"] == asset_new["externalId"]
 
-            # Path("https://...").is_file() crashes on windows with python 3.7
-            if not asset_src["content"].startswith("http") and Path(asset_src["content"]).is_file():
+            if not asset_src["content"].startswith("http") and os.path.isfile(asset_src["content"]):
```
Comment (PierreLeveau) [inline tests/e2e/test_copy_project.py:399]: we should leave the comment  to explain why we do not use `Path.is_file()`

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: tests/e2e/test_copy_project.py (hunk 1 above)
Comment (Jonas1312) [reply tests/e2e/test_copy_project.py:399]: os.path.is_file() works well on windows, is there a reason we shouldn't use it?

File: src/kili/services/export/format/kili/__init__.py [hunk 2]
```diff
@@ -43,12 +54,49 @@ def _save_assets_export(self, assets: List[Asset], output_filename: Path):
...
         self.make_archive(self.export_root_folder, output_filename)
 
         self.logger.warning(output_filename)
 
+    def _clean_filepaths(self, assets: List[Asset]):
+        """
+        Remove TemporaryDirectory() prefix from filepaths in "jsonContent" and "content" fields.
+        """
+        for asset in assets:
+            if asset["content"] != "" and os.path.isfile(asset["content"]):
```
[the remark this answers is not in the document]
Comment (Jonas1312) [reply src/kili/services/export/format/kili/__init__.py:69]: os.path.isfile() does the same thing? but has the advantage to work on windows unlike Path.is_file

File: tests/e2e/test_copy_project.py (hunk 1 above)
Comment (PierreLeveau) [reply tests/e2e/test_copy_project.py:399]: I agree, but someone might forget that there is a good reason not to use Pathlib, and will make the change the ther way around.

**Review by PierreLeveau — APPROVED** [review]

**Pull request merged**